blocks/blog-feature/blog-feature.js

```diff
@@ -0,0 +1,38 @@
+import { fetchIndex } from '../../scripts/scripts.js';
+
+export default async function decorate(block) {
+  const index = 'blogpost-index';
+  block.textContent = '';
+  // Fetch items from index
+  const indexContent = await fetchIndex(index);
+  if (indexContent.data.length > 0) {
+    const { image } = indexContent.data[0];
+    const { path } = indexContent.data[0];
+    const { title } = indexContent.data[0];
+    const { description } = indexContent.data[0];
+    // const { category } = indexContent.data[0];
+    const { pathname } = new URL(image, window.location.href);
+
+    const blogGrid = document.createRange().createContextualFragment('<div class="blog-feature-container"></div>');
+    const card = document.createRange().createContextualFragment(`
+           <div class="blog-img">
+           <picture>
```

Comment:
You can generate this using https://github.com/hlxsites/wilson/blob/main/scripts/lib-franklin.js#L398 and then also add the correct breakpoints for the width of `.blog-img`.

Reply:
Make sure the largest contentful paint (LCP) which in this case is the image of the blog feature is loaded with `eager=true`.

Here's an example how to use it:

```
block.querySelectorAll('img').forEach((img) => img.closest('picture').replaceWith(createOptimizedPicture(img.src, img.alt, true, [{ width: '1050' }])));
```